content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -180,52 +196,54 @@ A few limitations of that approach include non-trivial logic when dealing with c
 
 In order to avoid potential limitations as described above, clients may request the Table representation of objects, delegating specific details of printing to the server. The Kubernetes API implements standard HTTP content type negotiation: passing an `Accept` header containing a value of `application/json;as=Table;g=meta.k8s.io;v=v1beta1` with a `GET` call will request that the server return objects in the Table content type.
 
-For example:
+For example, list all of the pods on a cluster in the Table format.
 
-1. List all of the pods on a cluster in the Table format.
-
-        GET /api/v1/pods
-        Accept: application/json;as=Table;g=meta.k8s.io;v=v1beta1
-        ---
-        200 OK
-        Content-Type: application/json
-        {
-            "kind": "Table",
-            "apiVersion": "meta.k8s.io/v1beta1",
-            ...
-            "columnDefinitions": [
-                ...
-            ]
-        }
+```console
+GET /api/v1/pods
+Accept: application/json;as=Table;g=meta.k8s.io;v=v1beta1
+---
+200 OK
+Content-Type: application/json
+{
+    "kind": "Table",
+    "apiVersion": "meta.k8s.io/v1beta1",
+    ...
+    "columnDefinitions": [
+        ...
+    ]
+}
+```
 
 For API resource types that do not have a custom Table definition on the server, a default Table response is returned by the server, consisting of the resource's `name` and `creationTimestamp` fields.
 
-        GET /apis/crd.example.com/v1alpha1/namespaces/default/resources
-        ---
-        200 OK
-        Content-Type: application/json
-        ...
+```console
+GET /apis/crd.example.com/v1alpha1/namespaces/default/resources
+---
+200 OK
+Content-Type: application/json
+...
+{
+    "kind": "Table",
+    "apiVersion": "meta.k8s.io/v1beta1",
+    ...
+    "columnDefinitions": [
+        {
+            "name": "Name",
+            "type": "string",
+            ...
+        },
         {
-            "kind": "Table",
-            "apiVersion": "meta.k8s.io/v1beta1",
+            "name": "Created At",
+            "type": "date",
             ...
-            "columnDefinitions": [
-                {
-                    "name": "Name",
-                    "type": "string",
-                    ...
-                },
-                {
-                    "name": "Created At",
-                    "type": "date",
-                    ...
-                }
-            ]
         }
+    ]
+}
```

Comment:
To reviewers: the above lines are simply about removing unnecessary indentation. Obviously, 'git' gets confused about the change.